EdFi.Buzz.UI.Angular/src/app/Features/UploadSurvey/uploadSurveyWrapperComponent.tsx

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  OnChanges,
+  OnDestroy,
+  Output,
+  SimpleChanges,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+
+import { UploadSurvey } from './uploadSurvey';
+import * as React from 'react';
+
+import * as ReactDOM from 'react-dom';
+import { ApiService } from 'src/app/Services/api.service';
+
+const containerElementName = 'uploadSurveyReactComponentContainer';
+
+import GlobalFonts from '../../../globalstyle';
+import { ThemeProvider } from 'styled-components';
+import BuzzTheme from '../../../buzztheme';
+import { FormBuilder } from '@angular/forms';
+import { Title } from '@angular/platform-browser';
+import { Router, ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-upload-survey',
+  template: `<span #${containerElementName}></span>`,
+  styleUrls: ['./uploadSurvey.css'],
```

Comment:
I would suggest moving the styles into the UploadSurvey component. You could create a styled component for the most parent div and put all classes inside it. TeacherLanding has an example of this usage.

Reply:
Styles will be updated in FIF-242